andela/ah-titans
# #159431016 Integrate heroku

#### What does this PR do?
Seeks to merge the Heroku integration to develop branch.

#### Description of tasks to be completed
Add continuous integration with Heroku.

#### How should this be manually tested?

>Open your postman and send requests to various endpoints as described on the README.
>You should add the host as "ah-titans-api.herokuapp.com", without quotes, before the endpoints.
>You should be able to see results as shown on the screenshots below.

<img width="976" alt="screen shot 2018-08-02 at 17 30 51" src="https://user-images.githubusercontent.com/27728464/43592383-19d7a9d4-967e-11e8-8da0-b68e57099a35.png">

<img width="976" alt="screen shot 2018-08-02 at 17 29 47" src="https://user-images.githubusercontent.com/27728464/43592402-26a94262-967e-11e8-8482-140870f954c9.png">


 ###Any background context you want to provide?
Continuous deployment is enabled for the develop branch.

#### What are the relevant Pivotal Tracker Stories?
[#159431016](https://www.pivotaltracker.com/story/show/159431016)
